### DISCUSSION TITLE
https://nanaeo.cn/2023/12/30/2023end/

### DISCUSSION THREAD

**Discussion by giscus** [MliKiowa/Mlikiowa.github.io#12] [General]
# https://nanaeo.cn/2023/12/30/2023end/

It is a little blog～ - Mlikiowa - Village

https://nanaeo.cn/2023/12/30/2023end/

**Comment by xxhzm** [comment]
2025即将到来，大佬一年没更新啦

**Reply by MliKiowa** [reply]
在忙哦

**Comment by Cooooing** [comment]
2025即将到来，大佬一年没更新啦

**Reply by MliKiowa** [reply]
干什么干什么！

**Comment by Mitpoppy** [comment]
2025已经到来，大佬一年没更新啦
